workspaces/bulk-import/plugins/bulk-import-backend/src/service/handlers/import/execute-template.ts

```diff
@@ -0,0 +1,367 @@
+/*
+ * Copyright Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  AuthService,
+  DiscoveryService,
+  LoggerService,
+} from '@backstage/backend-plugin-api';
+import { Config } from '@backstage/config';
+
+import { fetchEventSource } from '@ai-zen/node-fetch-event-source';
```

Comment:
As mentioned in the package.json. I think this isn't the "right" import.

Reply:
See https://issues.redhat.com/browse/RHDHBUGS-2110 and https://issues.redhat.com/browse/RHDHBUGS-2111